docs/world api.md

```diff
@@ -702,6 +712,26 @@ def generate_output(self, output_directory: str):
     generate_mod(src, out_file, data)
 ```
 
+### Slot Data
+
+If the game client needs to know information about the generated seed, a preferred method of transferring the data
+is through the slot data. This can be filled in the `fill_slot_data` method on your world. This can be used to fill any
+data you need to know client side, and should return a `Dict[Any]`, but should be limited to only data that is
+absolutely necessary to not waste data. Slot data is sent to your client once it has successfully
+[connected.](network%20protocol.md#connected) If you need to know information about locations in your world, instead
```

Comment:
```suggestion
is through the slot data. This can be filled from the `fill_slot_data` method of your world by returing a `Dict[Any]`,
but should be limited to data that is absolutely necessary to not waste resources. Slot data is sent to your client once
it has successfully [connected](network%20protocol.md#connected).
If you need to know information about locations in your world, instead
```

Reply:
returing

Reply:
it's technically `Dict[Any, Any]` we just usually use `Dict[str, Any]`

Reply:
fixed the suggestion. Can't be [Any, Any] because of json

Reply:
The slot data needs to be JSON serializable.
The typing of JSON data has been discussed at length in other places https://github.com/python/typing/issues/182
Because of its complexity, I think it would be better to not mention the type and just mention that it needs to be JSON serializable.